projects/github-actions/repo-gardening/src/tasks/triage-issues/index.js

```diff
@@ -100,7 +149,12 @@ async function triageIssues( payload, octokit ) {
 		}
 
 		// Use OpenAI to automatically add labels to issues.
-		await aiLabeling( payload, octokit );
+		const issueLabels = await aiLabeling( payload, octokit );
+
+		// At this point, if we still miss a [Type] label, a [Feature] label, or a [Pri] label, ask the author to add it.
+		if ( ! issueLabels.some( label => /^\[(Type|Feature|Pri)\]/.test( label ) ) ) {
```

Comment:
The intent seems to be that we have at least one Type, one Feature, **and** one Pri, but the check is for having at least one Type, Feature, **or** Pri.

Reply:
Ah yes, you're right. I pushed f87d072ab9a1016a4f81a553ac50bbc7cb900cbe to fix that.